week-3/quote-generator/style.css

```diff
@@ -1 +1,50 @@
 /** Write your CSS in here **/
+
+:root {
+    --yellow:rgb(244, 168, 60);
+
+}
+
+body {
+    background-color:var(--yellow);
+}
+
+.main {
+    background-color: white;
+    color: var(--yellow);
+    font-size: 3rem;
+    margin: 7rem;
+    padding: 4rem;
+    display: flex;
+    flex-direction: column;
+}
+span{
+    font-size: 7rem;
+}
+.quote-line{
+    display: flex;
+}
+#author, #new-quote {
+    align-self: end;
+}
+button {
+    background-color: var(--yellow);
+    border: none;
+    color: white;
+    padding: 1.2rem;
+    font-size: 1.5rem;
+}
+
+.switch {
+    align-self: end;
+    font-size: 2rem;
+    padding-top: 1.5rem;
+}
+
+#auto-play-label {
+    font-size: 2rem;
+}
+
+.switch input {
+    transform: scale(1.5);
```

Comment:
I love what you've done with the CSS here - the use of flex and CSS vars is especially impressive and it looks great too!